### ISSUE TITLE
Return-value of byproducts in a link-file should be treated as an int

### ISSUE THREAD

**Issue by Xynnn007** [in-toto/in-toto-rs#30]
In [in-toto spec](https://github.com/in-toto/docs/blob/master/in-toto-spec.md#44-file-formats-namekeyid-prefixlink), it says
```plaintext
At a minimum, the byproducts dictionary should have standard output (stdout), 
standard error (stderr) and return value (return-value), even if no values are filled in. 
The return value should be stored as integer value.
```
An legal example of byproducts field is like
```json
"byproducts": {
    "return-value":0,
    "stderr":"a foo.py\n",
    "stdout":""
}
```
Present in-toto-rs implementation treats byproducts as a `HashMap<String, String>`, thus make link file like
```json
...
"byproducts": {
    "return-value":"0",
    "stderr":"a foo.py\n",
    "stdout":""
}
...
```
should fix this

**Comment by SantiagoTorres** [comment]
Oh, nice catch!

**Comment by adityasaky** [comment]
Closed by #31
